src/zcl_abapgit_branch_overview.clas.abap

```diff
@@ -8,19 +8,20 @@ CLASS zcl_abapgit_branch_overview DEFINITION
   PUBLIC SECTION.
 
     INTERFACES zif_abapgit_branch_overview .
+    TYPES: ty_commits TYPE STANDARD TABLE OF zif_abapgit_definitions=>ty_commit WITH DEFAULT KEY .
```

Comment:
changes to this include should not be part of this PR?

Reply:
No - messed up my repos...